torchx/schedulers/ray_scheduler.py

```diff
@@ -132,7 +144,44 @@ def run_opts(self) -> runopts:
         return opts
 
     def schedule(self, dryrun_info: AppDryRunInfo[RayJob]) -> str:
-        raise NotImplementedError()
+        # TODO: What to do with cfg
+        cfg: RayJob = dryrun_info.request
+
+        # TODO: figure out how to use
+        # app_id not needed here - ray will return a job_id
+        app_id: str
+
+        # no cluser config is needed since users will call ray up
+        cluster_config_file: str
```

Comment:
@amogkam Do you know how we will be able to retrieve the URL of the Job REST API without using a cluster YAML file? Right now the Ray CLI relies on the provided YAML file to figure out which cluster to use for a command. How is that going to work with the new Job API? My thinking was something along the line:

```
$ ray get-job-server-url <cluster_config_file>
```

following the idiom of the other Ray commands. Ideally with an accompanying API in the Ray Autoscaler SDK (e.g. `get_job_server_url()` that offers the same functionality programmatically.

Reply:
The cluster yaml file is what we use for ray to differentiate between different clusters.

We do have something like this already
either `ray get-head-ip <cluster yaml>` or in python `ray.autoscaler.sdk.get_head_ip("my_cluster.yaml")`